samples/package.json

```diff
@@ -6,32 +6,31 @@
   "author": "Google Inc.",
   "repository": "googleapis/nodejs-logging",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6.0.0"
   },
   "scripts": {
-    "cleanup": "repo-tools exec -- node scripts/cleanup",
-    "ava": "ava -T 20s --verbose test/*.test.js system-test/*.test.js",
-    "cover": "nyc --reporter=lcov --cache ava -T 20s --verbose system-test/*.test.js && nyc report",
-    "test": "repo-tools test run --cmd npm -- run cover"
+    "cleanup": "node scripts/cleanup",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
+    "test": "nyc mocha test/ system-test/ --timeout 20000"
   },
   "dependencies": {
-    "@google-cloud/logging": "1.2.0",
-    "@google-cloud/logging-bunyan": "0.8.2",
-    "@google-cloud/logging-winston": "0.9.0",
-    "@google-cloud/storage": "1.7.0",
-    "bunyan": "1.8.12",
-    "express": "4.16.3",
-    "fluent-logger": "2.7.0",
-    "winston": "2.4.0",
-    "yargs": "11.0.0"
+    "@google-cloud/logging": "^1.2.0",
```

Comment:
I think carets were not used by design so that samples always used fixed versions, @jmdobry knows more.

Reply:
That's what lock files are for.